data.md

```diff
@@ -413,9 +413,9 @@ A cons-list is used for the EVM wordstack.
 ```k
     syntax WordStack ::= #take ( Int , WordStack ) [function, functional]
  // ---------------------------------------------------------------------
-    rule #take(N, WS)         => .WordStack                      requires notBool N >Int 0
-    rule #take(N, .WordStack) => 0 : #take(N -Int 1, .WordStack) requires N >Int 0
-    rule #take(N, (W : WS))   => W : #take(N -Int 1, WS)         requires N >Int 0
+    rule [take.base]:      #take(N, WS)         => .WordStack                      requires notBool N >Int 0
+    rule [take.zero-pad]:  #take(N, .WordStack) => 0 : #take(N -Int 1, .WordStack) requires N >Int 0
+    rule [take.recursive]: #take(N, (W : WS))   => W : #take(N -Int 1, WS)         requires N >Int 0
```

Comment:
why are we labeling these rules?

Reply:
ah, this has nothing to do with the #abiEvenLog fix.  Just pushed together here because the change is small.  Let me know if you want me to have a separate PR.

Reply:
Well if we don't have a need for these labels, I'd rather remove them altogether.

Reply:
Ah, my previous answer wasn't clear. This is needed to fix another issue. I can make a separate PR for it.

Reply:
It is needed to refer to it in the --concrete-rules option of the java backend (recently added).  It turned out that the [concrete] attribute has a limitation, especially when multiple backends and multiple proofs want to have a different set of concrete rules.  For the deposit contract verification, I need to have one of the #take rule to be concrete, but if I just put the [concrete] attribute to it, it may be problematic for other proofs.  Indeed, the [concrete] attributes of the #signed rules are problematic for the deposit contract proofs.  So, the idea is to add labels for such rules, and allow the backend to select which rules to be treated as concrete rules.  This way, we can fix the rule duplication problem in the VSC repo, regarding the gas rules, which I plan to do next.